### PR TITLE
WIP: Icon layer demo

### PR DESCRIPTION
**Pull request by thomasgwatson**
@lorenjohnson perhaps with your experience you'll know how to reliably load images into the application? I have boiled down my issues to the simplest scenario I can in this example (this is not intended to be merged)

One from the webpack example, one from the branch I'm working on:
Screen Shot 2020-06-02 at 17 44 23
Screen Shot 2020-06-02 at 17 44 14

Getting 404 for the image, via storybook (webpack) but also via the regular dev setup (the main challenge). Aside of this tiny-demo, I have tried to emulate the utils/assets setup for a new graphic, tried relative paths, absolute paths...  I have not yet dived into the build system and attempted to grok that. 

I can get the graphic to load in the app via an img tag however...
Screen Shot 2020-06-02 at 17 46 08

Example code taken from https://github.com/visgl/deck.gl/tree/8.1-release/examples/website/icon
Docs here: https://deck.gl/#/documentation/deckgl-api-reference/layers/icon-layer

Anticipating that the map will be filled with `marker-100` markers from the icon graphic.

### CODE REVIEW

File: src/components/Map/layers/clusterLayer/index.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        coordinates: [parseFloat(post.location.center.lng), parseFloat(post.location.center.lat)]
+      }
+    }), onHover, onClick, getPosition: d => d.coordinates})
+}
+
+function getIconSize (size) {
+  return Math.min(100, size) / 100 + 1
+}
+
+// TODO: do a post filter function; map post type to color
```
Comment (tibetsprague) [inline src/components/Map/layers/clusterLayer/index.js:27]: i already created this: import { POST_TYPES } from 'store/models/Post'

File: src/components/Map/layers/clusterLayer/index.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  //         .map(f => f.properties)
+  //     }
+  //     info.object = pickedObject
+  //   }
+  //   return info
+  // }
+
+  renderLayers () {
+    const { data } = this.state
+    const { iconAtlas = 'public/iconAtlas.png',
```
Comment (tibetsprague) [inline src/components/Map/layers/clusterLayer/index.js:72]: did you try taking off the public part? throughout the code it looks like it is often just loading images by doing /image.png

Comment (tibetsprague) [reply src/components/Map/layers/clusterLayer/index.js:72]: this seems to work

Comment (thomasgwatson) [reply src/components/Map/layers/clusterLayer/index.js]: And it works, woop

**Pull request closed without merging**